src/util/artifacts.js

```diff
@@ -12,7 +12,8 @@ export async function prepareArtifactsDir(
   try {
     const stats = await fs.stat(artifactsDir);
     if (!stats.isDirectory()) {
-      throw new WebExtError(`${artifactsDir} is not a directory`);
+      throw new UsageError(`artifacts-dir=${artifactsDir} 
+                          (this value is not a directory)`);
```

Comment:
this line should not have so much whitespace in it because it will print like that in the console which will be harder to read. You can get around the line length limit in a couple ways. Like this:
````
throw new UsageError(
  `artifacts-dir=${artifactsDir} (this value is not a directory)`);
````
or like:
````
throw new UsageError(`artifacts-dir=${artifactsDir} ` +
                     `(this value is not a directory)`);
````